lib/acts_as_ar_query.rb

```diff
@@ -0,0 +1,199 @@
+require 'forwardable'
+# allow active record dsl to call legacy find
+class ActsAsArQuery
+  extend Forwardable
+  attr_accessor :klass, :mode, :options
+
+  # - [ ] bind
+  # - [ ] create_with
+  # - [ ] distinct
+  # - [ ] eager_load
+  # - [X] except ? - is this not defined in the interface?
+  # - [ ] extending
+  # - [ ] from
+  # - [ ] ~~group~~ - NO
+  # - [ ] ~~having~~ - NO
+  # - [X] includes (partial)
+  # - [ ] joins
+  # - [X] limit
+  # - [ ] lock
+  # - [.] none
+  # - [X] offset
+  # - [X] order (partial)
+  # - [ ] preload
+  # - [ ] readonly
+  # - [X] references (partial)
+  # - [X] reorder
+  # - [ ] reverse_order
+  # - [X] select (partial)
+  # - [X] unscope
+  # - [ ] uniq
+  # - [X] where (partial)
+  # - [ ] where.not
+
+  # - [X] all
+  # - [X] count
+  # - [ ] find
+  # - [X] first
+  # - [X] last
+  # - [X] size
+  # - [X] take
```

Comment:
:+1: Love the checklist, but can you put this latter half at the top?  Those are the more relevant ones.